package.json

```diff
@@ -4,7 +4,8 @@
     "description": "A system that allows efficient, cost-effective management of our library",
     "main": "index.js",
     "scripts": {
-        "start": "node server/index.js",
+        "build": "babel ./server -d build",
+        "start": "babel  ./server -d build && node build/index.js",
```

Comment:
You can use run build instead of the duplicate statements

Reply:
The correction has been made please review this again.